src/Entities/Project.php

```diff
@@ -3,6 +3,8 @@
 namespace ProjectManagementApi\Entities;
 
 use DateTime;
+use InvalidUserArrayDatatypeException;
```

Comment:
the namespace here is not what we expected - according to the diagram this should be ProjectManagementApi\Exceptions\InvalidUserArrayDatatypeException